airbyte-server/src/main/java/io/airbyte/server/handlers/WorkspacesHandler.java

```diff
@@ -176,6 +181,22 @@ public WorkspaceRead updateWorkspace(WorkspaceUpdate workspaceUpdate) throws Con
     return buildWorkspaceReadFromId(workspaceUpdate.getWorkspaceId());
   }
 
+  public NotificationRead tryNotification(Notification notification) {
+    try {
+      final NotificationClient notificationClient = JobNotifier.createNotificationClient(NotificationConverter.toConfig(notification));
+      final String message = String.format("Hello World! This is a test from Airbyte to try %s notification settings",
+          notification.getNotificationType());
+      if (notificationClient.notify(message)) {
+        return new NotificationRead().status(StatusEnum.SUCCEEDED);
+      }
+    } catch (IllegalArgumentException e) {
```

Comment:
why are you catching this one?

Reply:
I was so sure if this needed, how are the exceptions potentially translated into 404 messages such as declared in the config file here? 

 https://github.com/airbytehq/airbyte/blob/db119d043b3244009ef0eb54830f18a01a59ed87/airbyte-api/src/main/openapi/config.yaml#L412

I thought it might be because we throw them as KnownException with the statuscode but not quite sure about this...